package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "scripts": {
     "start": "nodemon src/app.js",
-    "test": "export IS_JEST=1 && yarn validate && jest --passWithNoTests",
+    "test": "set IS_JEST=1 && yarn validate && jest --passWithNoTests",
```

Comment:
@mkkasem if you look at the discussion between me and Shrreya on the https://github.com/RCDD-202110-TUR-BEW/backend-capstone-turkey-paying-it-forward/pull/23 PR you will get a better view on this.
set doesn't work in github, you need to use it locally but when you commit make sure that it is export not set
Just when you are going to push them to github after committing you can revert it back to set so it works on you terminal

Reply:
I saw the conversation and I know that I need to set it back to export, which I did in the last commit because I wasn't able to commit , so I fixed it on github immediately but I didn't know that if I change it to export without commiting it could work. 
Thanks ammar

Reply:
You are welcome 😊 
I faced the same problem at first, then I noticed that when you are pushing (as you know it runs the test script), it uses the script that is present in your local repo despite what is present in the commit.